app/test/marionette/lib/views/home.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var _ = require('lodash'),
+  View = require('./view'),
+  Question = require('./question');
+
+function Home() {
+  View.apply(this, arguments);
+}
+module.exports = Home;
+
+Home.prototype = {
+  __proto__: View.prototype,
+
+  openAskQuestion: function() {
+    var self = this;
+    return new Promise(function(resolve, reject) {
+      self.client.findElement(self._SELECTORS.ASK_QUESTION_BUTTON).click();
+      var question = new Question(self.client);
+      question.switchToFrame().then(function() {
+        resolve(question);
+      })
```

Comment:
`return question.switchToFrame()` won't return the `Question` page object, and we need it for the next step of the test :(